README.md

```diff
@@ -75,6 +75,9 @@ SLACK_TEAM_ID
 
 # Secret code required when redeeming .tech domains
 TECH_DOMAIN_REDEMPTION_SECRET_CODE
+
+# Who to assign Streak tasks to
+DEFAULT_TASK_ASSIGNEE
```

Comment:
What if this was called `DEFAULT_STREAK_TASK_ASSIGNEE_EMAIL`? Or just `DEFAULT_STREAK_TASK_ASSIGNEE`?